package.json

```diff
@@ -15,11 +15,10 @@
     "test:unit": "mocha --config test/unit/.mocharc.js",
     "test:integration": "mocha --config test/integration/.mocharc.js",
     "test:legacy": "mocha --config test/legacy/.mocharc.js",
-    "test:node": "npm run test:unit && npm run test:integration && npm run test:legacy",
+    "test:node": "npm run lint && npm run test:build && npm run test:unit && npm run test:integration && npm run test:legacy",
```

Comment:
Why add the linting here? `test:node` as I understanding is just meant as a quick test run.

Reply:
I removed it and pushed the changes to re-trigger the jobs

Reply:
Good shout!